static/app/routes.tsx

```diff
@@ -2103,6 +2103,22 @@ function buildRoutes() {
       <IndexRoute component={errorHandler(OverviewWrapper)} />
       <Route path="views/:viewId/" component={errorHandler(OverviewWrapper)} />
       <Route path="searches/:searchId/" component={errorHandler(OverviewWrapper)} />
+
+      {/* Redirects for legacy tags route. */}
+      <Redirect from=":groupId/tags/" to={`/issues/:groupId/distributions/`} />
+      <Redirect
+        from=":groupId/tags/:tagKey/"
+        to={`/issues/:groupId/distributions/:tagKey/`}
+      />
+      <Redirect
+        from={`:groupId/${TabPaths[Tab.EVENTS]}:eventId/tags/`}
+        to={`/issues/:groupId/${TabPaths[Tab.EVENTS]}:eventId/distributions/`}
+      />
+      <Redirect
+        from={`:groupId/${TabPaths[Tab.EVENTS]}:eventId/tags/:tagKey/`}
+        to={`/issues/:groupId/${TabPaths[Tab.EVENTS]}:eventId/distributions/:tagKey/`}
+      />
```

Comment:
@scttcper FYI this was a pain to sort out. 

Over in groupDetails we're returning `<GroupEventDetails />` without passing children in, so if we try to put any routes inside the `<Route path=":groupId/" ...>` then they get lost and don't take effect. But only when the conditions are right and line ~702 executes.

IMO we should always _always_ be including `children` from top level views like this. If there's a rule to distinguish `views/` from `components/` that might be it. Digging through such a huge view file to sort out what was happening took too many eng-hours :(

https://github.com/getsentry/sentry/blob/f34e5fff57b6bd8aa22dc54520fb285bef789cd8/static/app/views/issueDetails/groupDetails.tsx#L699-L705

Reply:
this is to support the old ui and the new ui at the same time. Can't support both easily from the router. In the new experience we want them to be on the /tags route and display the group details in the background. In the old experience we want the tags to be the only thing displayed